plugins/defaultplugins/ifplugin/vppcalls/afpacket_vppcalls_test.go

```diff
@@ -18,13 +18,17 @@ import (
 	"testing"
 
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/common/bin_api/af_packet"
+	interfaces2 "github.com/ligato/vpp-agent/plugins/defaultplugins/common/bin_api/interfaces"
```

Comment:
For clarity this should be named something like `binapi_interfaces` or maybe even rather rename `../common/model/interfaces` to `intf` to make it consistent with [other code](https://github.com/ligato/vpp-agent/blob/pantheon-dev/plugins/defaultplugins/ifplugin/afpacket_config.go#L24).